src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -1312,7 +1319,7 @@ protected void CreateHelpersForSpecifiedSSHComputerNames()
 
             foreach (string computerName in ResolvedComputerNames)
             {
-                var sshConnectionInfo = new SSHConnectionInfo(this.UserName, computerName, this.KeyFilePath);
```

Comment:
These creation helper functions are used by Invoke-Command, and this cmdlet will also have to be updated to support the new "Port" parameter.  All cmdlets that currently support SSH parameter sets are New-PSSession, Enter-PSSession, Invoke-Command.  You covered the first two but missed the last one.